appcelerator/titanium_mobile
# [TIMOB-25839] Android/iOS: Modified touch events to use default units defined in "tiapp.xml"

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-25839

**Summary:**
- This is a breaking-change.
- Android touch coordinates used to be hard-coded to pixels.
- iOS touch coordinates used to be hard-coded to dips.

**Test:**
_Run the below test on both Android and iOS._
1. Create a classic app using the test code attached to [TIMOB-25839](https://jira.appcelerator.org/browse/TIMOB-25839).
2. Set "tiapp.xml" property "ti.ui.defaultunit" to `dp`.
3. Run the project.
4. Drag your finger within the light-gray rectangle.
5. Verify that a green square is centered under your finger while you drag.
6. Set "tiapp.xml" property "ti.ui.defaultunit" to `px`.
7. Repeat steps 3-5.
